.github/workflows/ubuntu_workflow.yml

```diff
@@ -77,5 +77,4 @@ jobs:
           echo "**Timestamp:** ${{ steps.current-time.outputs.formattedTime }}" >> $GITHUB_STEP_SUMMARY
           echo "**Buildscan URL for ubuntu-workflow run:**" >> $GITHUB_STEP_SUMMARY
           echo "[${{ github.run_id }}](https://github.com/Flank/flank/actions/runs/${{ github.run_id }})" >> $GITHUB_STEP_SUMMARY
-          echo "${{ needs.build.outputs.build-scan-url }}" >> $GITHUB_STEP_SUMMARY
-      
\ No newline at end of file
+          echo "${{ needs.build.outputs.build-scan-url }}" >> $GITHUB_STEP_SUMMARY
```

Comment:
This should have been changed to use an `env:`, and you're missing the blank line at the end of the file.

Reply:
Oops. Can you send a PR? Thanks!

Reply:
Sure

Reply:
https://github.com/Flank/flank/pull/2547